build.gradle.kts

```
@@ -14,6 +15,28 @@ plugins {
 
 repositories {
     mavenCentral()
+
+    maven {
+        name = "Confluent maven repo"
+        url = uri("https://packages.confluent.io/maven/")
+    }
+
+    val githubToken = System.getenv("GITHUB_TOKEN")
+    if (githubToken.isNullOrEmpty()) {
+        maven {
+            name = "internal-mirror-github-navikt"
+            url = uri("https://repo.adeo.no/repository/github-package-registry-navikt/")
+        }
```

`repo.adeo.no` krever naisdevice for å kunne brukes. Så kunne vi gjort noe sånt i stedet?
```suggestion
        maven {
            name = "external-mirror-github-navikt"
            url = uri("https://github-package-registry-mirror.gc.nav.no/cached/maven-release")
        }
```